pkg/configurator/crd_client.go

```diff
@@ -43,8 +103,7 @@ func newConfiguratorWithCRDClient(meshConfigClientSet versioned.Interface, stop
 	// start listener
 	go crdClient.runMeshConfigListener(stop)
 
-	// start informer
-	go informer.Run(stop)
+	crdClient.run(stop)
```

Comment:
Is the change to drop the goroutine intentional? If yes, could you make this in a separate change with more context?

Reply:
No, the goroutine invocation is moved into `crdClient.run()`

https://github.com/openservicemesh/osm/blob/c73d4b0a0195cab7ec5d1b1aac2d488ff5d52efa/pkg/configurator/crd_client.go#L149